company_sanitary_registry/models/res_company.py

```diff
@@ -6,4 +6,4 @@
 class ResCompany(models.Model):
     _inherit = "res.company"
 
-    sanitary_registry = fields.Char()
```

Comment:
What do you think about maintain this field as related to first sanitary registry to minimize initial fails in other modules that still uses the old field?